internal/notifier/azure_devops.go

```diff
@@ -172,3 +179,12 @@ func duplicateAzureDevOpsStatus(statuses *[]git.GitStatus, status *git.GitStatus
 
 	return false
 }
+
+func commitStatusGenre(event eventv1.Event) string {
+	summary, ok := event.Metadata["summary"]
+	if ok {
+		return fmt.Sprintf("%s:%s", genre, summary)
```

Comment:
Does Azure DevOps supports free-form text as genre? The summary can be 255 in length, is that supported?

Reply:
Doesn't say anything about max length, but Ok it seems that adding summary there could work.